recipe/meta.yaml

```
@@ -23,36 +23,46 @@ requirements:
   run:
     - python >=3.6
     - numpy
-    - scipy
+    - scipy >=1.2.0
     - scikit-image
-    - vispy
-    - dask
-    - cachey
+    - vispy >=0.6.4
+    - dask[array] >=2.1.0
+    - cachey >=0.2.1
     - zarr
     - fsspec
-    - imageio
-    - ipykernel
+    - imageio >=2.5.0
+    - importlib-metadata >=1.5.0
+    - ipykernel >=5.1.1
+    - IPython >=7.7.0
     - jupyter_client
-    - qtpy
-    - qtconsole
+    - qtpy >=1.7.0
+    - qtconsole >=4.5.1
     - IPython
     - backcall
+    - Pillow !=7.1.0,!=7.1.1  # not a direct dependency, but 7.1.0 and 7.1.1 broke imageio
     - pluggy
-    - PyOpenGl
+    - psutil >=5.0
+    - PyOpenGL >=3.1.0
     - PySide2
```

Would lower case this as well (even though it precedes this PR). Might as well get all of this updated :)

```suggestion
    - pyside2
```